include/mbedtls/mbedtls_config.h

```diff
@@ -3073,6 +3084,85 @@
 
 /** \} name SECTION: mbed TLS modules */
 
+/**
+ * \name SECTION: General configuration options
+ *
+ * This section contains Mbed TLS build settings that are not associated
+ * with a particular module.
+ *
+ * \{
+ */
+
+/**
+ * \def MBEDTLS_CONFIG_FILE
+ *
+ * If defined, this is a header which will be included instead of
```

Comment:
Correct me if I'm wrong: in order for this to work properly, MBEDTLS_CONFIG_FILE has to be defined via command-line, and the value that is used here has then no meaning?

Reply:
Yes. That's a good point. When I wrote this, the context I had in mind was the HTML documentation generated by Doxygen. For people who are reading the header file directly, it might be worth stressing this point. I'll think about how to word it.